templates/etc/flapjack/flapjack_config.erb

```diff
@@ -283,7 +283,14 @@ production:
       port: <%=@prd_web_ui_port%>
       timeout: <%=@prd_web_ui_timeout%>
       access_log: '<%=@prd_log_dir-%><%=@prd_web_ui_accesslog-%>'
+<% if @prd_web_ui_base_url -%>
+      base_url: '<%=@prd_web_ui_base_url-%>'
+<% end -%>
```

Comment:
I would like to define a custom internal port for flapjack but not use this in public (similar to my comment above). I think this is not possible with your proposal or do I missunderstand the possibilities?